packages/core/admin/admin/src/translations/en.json

```diff
@@ -679,6 +679,7 @@
   "content-manager.apiError.This attribute must be unique": "{field} must be unique",
   "form.button.continue": "Continue",
   "form.button.done": "Done",
+  "global.Search": "Search",
```

Comment:
about using admin translations for plugins - after discussing it with the team, it sounds good indeed!

following up:
- could we use the same format as for others translations keys (all lowercase ➡️ `"global.search"`)?
- I think we could add this fix (#13533) into this PR, using the same key you've created. I'll close the other PR once this one is merged!

does that sound good to you?